foundry.toml

```diff
@@ -73,37 +73,38 @@ base = "${ETH_NODE_URI_BASE}"
 
 [etherscan]
 mainnet = { key = "${MAINNET_ETHERSCAN_API_KEY}", chain = "1" }
```

Comment:
I haven't checked that but the default url is: https://api.etherscan.io/api and doesn't have to be compatible with api key v2. I would add explicit url: https://api.etherscan.io/v2/api?chainid=1